eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantAccessioningDatabaseService.java

```diff
@@ -26,16 +27,17 @@
 import java.util.Collection;
 
 public class SubmittedVariantAccessioningDatabaseService
-        extends BasicSpringDataRepositoryDatabaseService<ISubmittedVariant, SubmittedVariantEntity, Long>
+        extends BasicSpringDataRepositoryDatabaseService<ISubmittedVariant, Long, SubmittedVariantEntity>
         implements MonotonicDatabaseService<ISubmittedVariant, String> {
 
-    public SubmittedVariantAccessioningDatabaseService(SubmittedVariantAccessioningRepository repository) {
+    public SubmittedVariantAccessioningDatabaseService(SubmittedVariantAccessioningRepository repository,
+                                                       BasicInactiveAccessionService inactiveAccessionService) {
```

Comment:
why the generics interface is not complete for BasicInactiveAccessionService?
I know it's ugly, but with all the changes that occurred, I think it can save a bug or two:
```
    public SubmittedVariantAccessioningDatabaseService(SubmittedVariantAccessioningRepository repository,
                                                       BasicInactiveAccessionService<Long,
                                                               SubmittedVariantEntity,
                                                               SubmittedVariantInactiveEntity,
                                                               SubmittedVariantOperationEntity
                                                               > inactiveAccessionService) {
```